bubo/bot_commands.py

```diff
@@ -609,3 +612,15 @@ async def _users(self):
         if not text:
             text = help_strings.HELP_USERS
         await send_text_to_room(self.client, self.room.room_id, text)
+    async def _keys(self):
```

Comment:
```suggestion
        await send_text_to_room(self.client, self.room.room_id, text)
        
    async def _keys(self):
```

Blank line between methods - https://pep8.org/#blank-lines